KsApi/Service.swift

```diff
@@ -92,8 +92,13 @@ public struct Service: ServiceType {
   }
 
   public func cancelBacking(input: CancelBackingInput)
-    -> SignalProducer<GraphMutationEmptyResponseEnvelope, GraphError> {
-    return applyMutation(mutation: CancelBackingMutation(input: input))
+    -> SignalProducer<EmptyResponseEnvelope, ErrorEnvelope> {
+    return GraphQL.shared.client
+      .perform(mutation: GraphAPI
+        .CancelBackingMutation(input: GraphAPI.CancelBackingInput(id: input.backingId)))
+      .flatMap { _ in
+        SignalProducer(value: EmptyResponseEnvelope())
```

Comment:
No need for an adapter here since we are simply returning the empty envelope, like before.